File: Java/ArraysBidimensionals/src/ex04.java
public class ex04 {

	public static void main (String[] args){

		char[][] matrix = new char[5][5];

		for (int i = 0; i < matrix.length; i++) {

			for (int j = 0; j < matrix[i].length; j++) {
				if ((i-j)<0){
					matrix[i][j]='-';
				} else if ((i-j)>0){
					matrix[i][j]='+';
				} else{
					matrix[i][j]='0';
				}
			} 
		} 
		int i = 0;
		int j = 0;

		while (i < matrix.length) {
			j = 0;

			while(j < matrix[i].length){

				System.out.print (matrix[i][j]);
				j++;
			}
			System.out.println ();
			i++;
		}

	}

}

File: Base de dades/Php/PhpProjectArray/ex04.php
<html>
    <head>
        <meta charset="UTF-8">
        <title>ex04</title>
    </head>
    <body>
        <?php
        //Calcula el máximo del array e indica en qué posición del array está.
        $numeroMayor = -1;
        $contador = 0;
        for ($i = 0; $i < 10; $i++) {
            $numero[$i] = rand(0, 100);
            count($numero);
            echo $numero[$i] . " ";
            if ($numero[$i] > $numeroMayor) {
                $numeroMayor = $numero[$i];
                $contador = count($numero);
            }
        }
        echo " numero mayor= " . $numeroMayor;
        echo " Posición array= " . $contador;
        ?>
    </body>
</html>

